show_graph.py

```diff
@@ -137,6 +204,12 @@ def main():
   parser.add_argument("--fieldname",
                       help="the name of the field for which you want the graph",
                       default="total_req_per_sec")
+  parser.add_argument("--arrangement", help=("the type of arrangement in"
+                                             " this experiment."),
```

Comment:
I think you need to better document in various places (and here) what an arrangement is.